portfolio/src/main/webapp/script.js

```diff
@@ -40,6 +40,60 @@ function getFromDataUsingArrow() {
   });
 }
 
+function getSelectAmountComments() {
+  var i;
+  fetch('/data').then(response => response.json()).then((data) => {
+    do {
+      var commentsShown = prompt("Enter a  number (1-10)");
+      commentsShown = parseInt(commentsShown,10);
+    } while (commentsShown < 1 || commentsShown > 10);
+    console.log("Inputted", commentsShown);
+    const commentElement = document.getElementById('comment-container');
+    commentElement.innerHTML = '';
+    if (data.length < commentsShown) {
```

Comment:
Another way to do this to reduce duplicated code is to grab the minimum of data.length and commentsShown.